md_core/add_rtt/r_column_bots/pup_table.py

```
@@ -0,0 +1,108 @@
+            add_from_redirect.append(title)
+        else:
+            no_add.append(title)
+    # ---
+    printe.output(f"<<yellow>> no_add: {len(no_add)}, already_in: {len(already_in)}")
+    # ---
+    printe.output(f"<<yellow>> add_done: {len(add_done)}, add_from_redirect: {len(add_from_redirect)}")
+    # ---
+    return table.string
+
+
+def add_rtt_to_tables(text, redirects={}, pages=[], table=False):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)